binary/script/shell.nix

```diff
@@ -1,27 +1,31 @@
-let _pkgs = import <nixpkgs> { };
-in { pkgs ? import (_pkgs.fetchFromGitHub {
-  owner = "NixOS";
-  repo = "nixpkgs";
-  #branch@date: nixpkgs@2020-11-24
-  rev = "6625284c397b44bc9518a5a1567c1b5aae455c08";
-  sha256 = "1w0czzv53sg35gp7sr506facbmzd33jm34p6cg23fb9kz5rf5c89";
-}) { } }:
-
-with pkgs;
-
 let
-
-in mkShell {
-  buildInputs = [
-    curl
-    expect
-    gcc
-    git
-    gnumake
-    go
-    perl
```

Comment:
Looks like `perl` is still needed. 
```bash
sh: line 1: perl: command not found
make: *** [Makefile.housekeeping:1221: bin-x86_64-efi/ipxe.efi.tmp] Error 127
rm bin-x86_64-efi/version.ipxe.efi.o
make: Leaving directory '/code/upstream-9062544f6a0c69c249b90d21a08d05518aafc2ec-ipxe.efi/src'
make: *** [Makefile:30: binary/ipxe.efi] Error 2
```

Reply:
oops I thought that was for `git` :man_facepalming: